.circleci/real_config.yml

```diff
@@ -391,12 +391,12 @@ commands:
                 command: |
                   sudo bash /tmp/install_python.sh <<parameters.python-version>>
 
-            - run:
-                name: Setup venv
-                command: |
-                  python3 -m venv /tmp/venv
-                  echo 'export PATH=/tmp/venv/bin:$PATH' >> $BASH_ENV
-                  /tmp/venv/bin/python -m pip install --upgrade pip wheel setuptools
+      - run:
```

Comment:
This is part of a job called `setup-python-venv`, and... used to be that you needed to set `install-python=true` to actually start up the virtual environment? That seems wrong, and this patch reads to me like what the config should have been in the first place.

Do you have any idea if that earlier behavior was intentional, or do you think this is just fixing a bug?

Reply:
I will need to go back and look through the history but it does indeed appear to be a bug. I could imagine it being disabled intentionally at some point, but more likely it was never updated when we no longer needed to use a conda install

Reply:
Ooh. That's a really good theory.

Reply:
Yeah it looks like an unfortunate side effect of this PR:
https://github.com/determined-ai/determined/pull/4535
cc @loksonarius